calblueprint/unloop
# Bulk upload action items

*You only need to review changes in `/controllers/api/assignments_controller`*. Allow for bulk upload of action items. Now you can bulk upload multiple assignments that correspond to multiple action items, whereas before you could only bulk upload multiple assignments that corresponded to a single action item. 

## To test:
Provided for testing convenience is `AssignmentButton` which is a react component that is placed in the endpoint `/assignments`. The assignments it creates/deletes is hardcoded, though, so if you would like to change what you are uploading (title/description) or who you upload to (staff, etc...) then make sure you change that in the component code in `components/AssignmentButton`. 

## THE COMPONENT AssignmentButton SHOULD BE DELETED BEFORE THE MERGE. It is only for testing and feel free to ignore it during your PR review.


CC: @didvi
